datalad/distribution/publish.py

```diff
@@ -600,6 +595,12 @@ class Publish(Interface):
       to publish a dataset. Publication targets are either configured remote
       Git repositories, or git-annex special remotes (if they support data
       upload).
+
+    ..note::
+      A new (since 0.13.0) `push` command provides an alternative interface to
+      publish to siblings. Critical differences from `publish` behavior are: `push`
+      by default copies annexed data, and it does not have `--missing` flag so you
+      will use `create-sibling`.
```

Comment:
From a comprehension standpoint, I struggled with the "you will use `create-sibling`" part.  Perhaps "need to" after "will" would make things clearer.  Otherwise, it reads fine to me and (AFAIU) is in line with what was decided on the call.

Here's a fuller and more subjective rewrite, in case you think any parts of it are an improvement:

```
      The `push` command (new in 0.13.0) provides an alternative interface.
      Critical differences are that `push` transfers annexed data by default
      and does not handle sibling creation (i.e. it does not have a `--missing`
      option).
```

Reply:
I like!

Reply:
thanks, will rewrite with this wording